### PR TITLE
feat(tests) - test WatchTickers &  test fetchTickers updates (single & multi symbol tests)

### PR DESCRIPTION
**Pull request by ttodua**

### CODE REVIEW

**Comment by carlosmiei** [comment]
@ttodua shouldn't we add the test to the tests object to it can run?

**Comment by ttodua** [comment]
> @ttodua shouldn't we add the test to the tests object to it can run?

1 sec

**Comment by ttodua** [comment]
right, test.ts was missing sorry, pushed.

**Comment by ttodua** [comment]
i'll test update on my side and lyk if PR ready.

**Comment by ttodua** [comment]
some issues, fixing now

File: ts/src/test/Exchange/base/test.sharedMethods.ts [hunk 1]
```diff
@@ -11,6 +11,10 @@ function isTemporaryFailure (e) {
     return (e instanceof OperationFailed) && (!(e instanceof OnMaintenance));
 }
 
+function isArgumentsRequiredException (e) {
```
Comment (carlosmiei) [inline ts/src/test/Exchange/base/test.sharedMethods.ts:14]: I don't think we need this method, encapsulates something very simple that we can do directly

Comment (ttodua) [reply ts/src/test/Exchange/base/test.sharedMethods.ts:14]: if i recall it correctly, the custom imports were specifically handled inside transpiler.js's `isSharedMethodTests` logic block, don't remind the reason. if it cant be left such, i'll revise it a bit later today

**Comment by ttodua** [comment]
@carlosmiei PR ready.

**Pull request merged**